packages/valory/skills/decision_maker_abci/behaviours/decision_request.py

```diff
@@ -82,6 +98,16 @@ def async_act(self) -> Generator:
             if not self.n_slots_supported:
                 mocking_mode = None
 
+            if self.benchmarking_mode.enabled:
+                # check if the bet_id_row_manager has been loaded already
+                if len(self.shared_state.bet_id_row_manager) == 0:
+                    bets_mapping = self.initialize_bet_id_row_manager()
+                    self.shared_state.bet_id_row_manager = bets_mapping
+                    log_message = (
+                        f"Loaded bets mapping: {self.shared_state.bet_id_row_manager}"
+                    )
+                    self.context.logger.info(log_message)
```

Comment:
This message can get too big and it's not truncated.

Reply:
this was just for debugging purposes. Removed.